modules/front-end/package.json

```diff
@@ -21,6 +21,7 @@
     "@angular/platform-browser-dynamic": "^14.0.0",
     "@angular/router": "^14.0.0",
     "@antv/g2": "^4.2.7",
+    "lodash-es": "^4.17.21",
     "lodash.differenceby": "^4.8.0",
```

Comment:
You can remove 
- lodash.differenceby
- lodash.get
- lodash.intersectionby
- lodash.keyby